data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,58 @@ mission "Remnant: Scent of Rain 1"
 			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
 			label scanningpayment
 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
+
+
+
+mission "Remnant: Wont Someone Please Think Of The Children"
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to fight and disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	source
+		government "Remnant"
+		not attributes "station"
+	waypoint "Antevorta"
+	to offer
+		and
+			"remnant: broken jump drive count" >= 10
+			or
+				and
+					has "Remnant: Cognizance 25: done"
+					not "flagship planet: Ssil Vida"
+				has "event: remnant: cognizance calm"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request for you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Kas'lor Ik 582 to be precise. Normally we would request you track down this ship and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can disarm the ship and fix their jump drive and hope they decide to head back home."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have no intentions of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Let's make sure everyone gets home safely."`
+				accept
+			label rathernot
+			`	He nods in acknowledgment. "I feared my ask was too much. I more than understand the sentiment. Maybe I was being naive to think such an outcome was possible."`
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Antevorta"
+		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"
```

Comment:
Just as context I took three of the name parts from the Korath definition file. So hopefully I understood how the name system works.

Reply:
Are you sure you were looking at the right file?  Because I can't find bits that would make that up.  If I refer to an older version of the file, however, this can be done:

```suggestion
		ship "Kas'lor Ik 582 (Crippled)" "Fligrookor"
```